docker-compose.yml

```diff
@@ -274,6 +274,31 @@ services:
         - /var/run/docker.sock:/var/run/docker.sock:ro
         - /lib/modules:/lib/modules:ro
 
+    acme-mailcow:
+      depends_on:
+        - nginx-mailcow
+      image: mailcow/acme
+      build: ./data/Dockerfiles/acme
+      # All domains to be included in the certificate
+      command: ${MAILCOW_HOSTNAME}
+      #autodiscover.${MAILCOW_HOSTNAME} autoconfig.${MAILCOW_HOSTNAME}
+      environment:
+        - CONTAINERS_RESTART=mailcowdockerized_postfix-mailcow_1 mailcowdockerized_dovecot-mailcow_1 mailcowdockerized_nginx-mailcow_1
+      volumes:
+        - ./data/web/.well-known/acme-challenge:/var/www/acme:rw
+        - ./data/assets/ssl:/var/lib/acme/:rw
+        - /var/run/docker.sock:/var/run/docker.sock:ro
+      dns:
+        - 172.22.1.254
+      dns_search: mailcow-network
+      # TODO: find some way to use always
+      # do not restart the container after exit. Things get worse when we hit let's encrypt's ratelimit.
+      restart: "no"
```

Comment:
Well, reading the docs again:

> If a container is successfully restarted (the container is started and runs for at least 10 seconds), the delay is reset to its default value of 100 ms.

I always had in my mind, this was at 1s instead of 10s. acme-client should fail in less than 10s. So this is safe to use with `on-failure:3` and not hit rate limits.

Thanks.